src/web/modules/gateway_accounts/switch_psp/switch_psp.http.ts

```diff
@@ -6,6 +6,19 @@ import logger from "../../../../lib/logger";
 
 import stripeTestAccount from '../../stripe/test-account.http'
 
+async function setConnectorStripeSetup(gatewayAccountId: string, isTest: boolean) {
```

Comment:
Can we name `isTest` to something meaningful?

Reply:
I suppose we could just pass the entire gateway account object to this function, I thought `isTest` was clear re "is the gateway account a test account"

Reply:
We could name it like `markTasksAsCompleted` so it relates to what the method is doing.

To go even further, it could be like

```
async function setConnectorStripeSetup(gatewayAccountId: string, operation: string) {
    const addOrRemove = (operation === 'remove')? false: true

    await Connector.accounts.updateStripeSetup(gatewayAccountId, {
          bank_account: addOrRemove,
          company_number: addOrRemove,
          responsible_person: addOrRemove,
          vat_number: addOrRemove,
          director: addOrRemove,
          organisation_details: addOrRemove,
          government_entity_document: addOrRemove
        })

}
```